docs/loading/Spark-connector-starrocks.md

```diff
@@ -303,31 +303,34 @@ Construct a streaming read of data from a CSV file and load data into the StarRo
         .getOrCreate()
```

Comment:
Actually, I verified all the examples when I wrote this doc. Are you sure that an error is returned when the following codes are executed? The use of backslashes \ is for line continuation, enhancing code readability by breaking long lines into more readable multiple lines. In this context, the backslashes have no impact on the functionality of the code.

Reply:
Yes, I'm sure, please just copy the original code to your IDE and see the prompts of the IDE. Especially the last right bracket, no matched left bracket before.